Source/Core/VideoCommon/PerformanceMetrics.cpp

```diff
@@ -16,6 +20,10 @@ void PerformanceMetrics::Reset()
   m_fps_counter.Reset();
   m_vps_counter.Reset();
   m_speed_counter.Reset();
+
+  m_time_sleeping = DT::zero();
+  m_real_times.fill(Clock::now());
+  m_cpu_times.fill(Core::System::GetInstance().GetCoreTiming().GetCPUTimePoint(0));
```

Comment:
`m_time_index` should be reset as well

Reply:
And `m_graph_max_time` ?

Reply:
`m_time_index` shouldnt matter because its a circular buffer, idk about m_graph_max_time, probably.. Ill do that tomorrow.

Reply:
`m_graph_max_time` should be ok because it converges in 2.5 seconds and its better if it doesnt start at 0